tests/10apidoc/02login.pl

```diff
@@ -132,7 +132,7 @@
    requires => [ $main::API_CLIENTS[0],
                  qw( can_login_password_flow )],
 
-   bug => "SYN-680",
+   # bug => "SYN-680", # now fixed
```

Comment:
If it's a true "regression test" for a known bug then it can live in `90jira/` named after the bug it asserts no longer exists. If it's just a test for generically specified behaviour then it's probably less important that it used to not work due to a bug.